src/prg_system_mod.F90

```diff
@@ -695,17 +710,24 @@ subroutine prg_write_system(system,filename,extension)
           resindex = system%resindex
        endif
 
+       if(.not.allocated(system%resname))then
+          allocate(resname(nats))
+          resname = "MOL"
```

Comment:
Same here, why set it to `MOL`?

Reply:
In this case I need it cause if by chance the system was read from .xyz format 
the the rename is not set. So, better to have something like "MOL"

Reply:
Ah, thanks, I missed that.